src/components/JobsTrainingButtons/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const JobsTrainingButtons = ({ setCardsArr, setSearchCards }) => {
+  const { t } = useTranslation();
+  //The selected button is disabled to let the user know which data he is viewing (jobs or training).
+  const [isDisabled, setIsDisabled] = useState(true);
+
+  // If a button is disabled it means it's data is already fetched. So, on the click of the other button this function detects the disabled button and fetches the opposite data.
+  const fetchOpportunityData = () => {
+    if (isDisabled) {
+      setSearchCards([...t(`home.trainings`, { returnObjects: true })]);
+      setCardsArr([...t(`home.trainings`, { returnObjects: true })]);
+      setIsDisabled(!isDisabled);
+    } else {
+      setSearchCards([...t(`home.jobOpportunities`, { returnObjects: true })]);
+      setCardsArr([...t(`home.jobOpportunities`, { returnObjects: true })]);
+      setIsDisabled(!isDisabled);
+    }
+  };
+
+  const buttonNames = ["Job Opportunities", "Trainings / Workshops"];
```

Comment:
move this as a const below the imports as mentioned, with BUTTON_NAMES or something like this.

Reply:
I tried moving it into line 3 under imports. it  gives this error:  Import in body of module; reorder to top  import/first

Reply:
doesn't sound right. there's other examples of code that do this in your codebase. 

https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/first.md

Reply:
double check where you moved it, or maybe your eslint was stale in saying it

Reply:
sorry my bad I placed it in line 3, I should have made it under it.

Reply:
know it's working